content/docs/06-integrations/00-kubernetes.md

```diff
@@ -411,14 +714,150 @@ The Kubeadm configuration file is where you can do the following:
+```
+
+<br />
+
+### Configure OIDC Identity Provider
+
+Platforms that use PXK can use the OIDC IDP feature. When you add the PXK pack to a cluster profile, Palette displays the following OIDC IDP options. 
+
+All the options require you to map a set of users or groups to a Kubernetes RBAC role. To learn how to map a Kubernetes role to users and groups, refer to [Create Role Bindings](/clusters/cluster-management/cluster-rbac/#createrolebindings). 
+
+You can create a role binding that maps individual users or groups assigned within the OIDC provider's configuration to a role. To learn more, review [Use RBAC with OIDC](/integrations/kubernetes#userbacwithoidc).
+
+<br />
+
+- **None**: This is the default setting and there is nothing to configure. This setting displays in the YAML file as `noauth`. 
+
+  <br />
+
+  <WarningBox>
+
+  We do not recommend choosing **None** in a production environment, as it may disable authentication for add-ons that rely on OIDC.
+
+  </WarningBox>
+
+- **Custom**: This setting allows you to specify a third-party OIDC provider by configuring OIDC statements in the Kubeadm configuration file as described in [Configure Custom OIDC](/integrations/kubernetes#configurecustomoidc). This setting displays in the YAML file as `none`.
+
+
+- **Palette**: This setting makes Palette the IDP. Any user with a Palette account in the tenant and the proper permissions to view and access the project's resources is able to log into the Kubernetes dashboard. This setting displays in the YAML file as `palette`.
+
+
+- **Inherit from Tenant**: This setting allows you to apply RBAC to multiple clusters and requires you to configure OpenID Connect (OIDC) in **Tenant Settings**. In Tenant Admin scope, navigate to **Tenant Settings** > **SSO**, choose **OIDC**, and provide your third-party IDP details. This setting displays in the YAML file as `tenant`. For more information, check out the [SSO Setup](/user-management/saml-sso) guide.
+
+<InfoBox>
+
+If your IDP uses Security Assertion Markup Language (SAML) authentication, then the **Inherit from Tenant** option will not work, and you will need to use the **Custom** option instead. This is because Kubernetes supports only OIDC authentication and not SAML authentication.
+
+</InfoBox>
+
+
+### Configure Custom OIDC
+
+The custom method to configure OIDC and apply RBAC for an OIDC provider can be used for all cloud services except Amazon Elastic Kubernetes Service (EKS) and [Azure-AKS](/clusters/public-cloud/azure/aks/#configureanazureactivedirectory).
+
+<br />
+
+<Tabs>
+
+<Tabs.TabPane tab="Custom OIDC Setup" key="Custom OIDC Setup">
+
+
+Follow these steps to configure a third-party OIDC IDP. You can apply these steps to all the public cloud providers except [Azure-AKS](/clusters/public-cloud/azure/aks/#configureanazureactivedirectory) and Amazon [EKS](/integrations/oidc-eks/) clusters. Azure AKS and Amazon EKS require different configurations.
```

Comment:
Two notes about this paragraph:

1. The EKS link is incorrect and will cause a broken URL. Where are you trying to take the user? TO the tab? If so, let's guide them to the tab.
2. Yes, AKS is different but let's help the reader by providing context. Because if you click on the URL you get dropped into an Azure AD page.

Reply:
```suggestion
Follow these steps to configure a third-party OIDC IDP. You can apply these steps to all the public cloud providers except Azure AKS and Amazon [EKS](/integrations/oidc-eks/) clusters. Azure AKS and Amazon EKS require different configurations. AKS requires you to use Azure AAD to enable OIDC integration. Refer to the [Configure an Azure Active Directory](/clusters/public-cloud/azure/aks/#configureanazureactivedirectory) to learn more.
```

Reply:
Fixed.